client/web/compose/src/components/PageBlocks/Navigation/Base.vue

```diff
@@ -0,0 +1,136 @@
+<template>
+  <wrap
+    v-bind="$props"
+    v-on="$listeners"
+  >
+    <div class="h-100 w-100 navigation-page-block-item">
+      <b-nav
+        :tabs="options.display.appearance === 'tabs'"
+        :pills="options.display.appearance === 'pills'"
+        :small="options.display.appearance === 'small'"
+        :fill="options.display.fillJustify === 'fill'"
+        :justify="options.display.fillJustify === 'justify'"
+        :align="options.display.alignment"
+        class="border-0 h-100"
+      >
+        <b-nav-item
+          v-for="(list, index) in options.lists"
+          v-show="list.options.itemOption.label"
+          :key="`lists-${index}`"
+          :to="processLink(list)"
+          :href="processHrefLink(list)"
+          :disabled="list.options.disabled"
+          :variant="list.options.bgVariant"
+          :style="{ order: index, color: list.options.textColor }"
+          class="nav-list-item"
```

Comment:
If you really need this custom class then try to write is as bootstrap class not by introducing a new one